macros/utils/dict_utils/undefined_dict_keys_to_none.sql

```diff
@@ -1,6 +1,6 @@
-{% macro empty_dict_keys_to_none(dict) %}
+{% macro undefined_dict_keys_to_none(dict) %}
```

Comment:
We only want to turn undefined -> none
but not other empty values (empty strings, empty lists, etc)